makeflow/src/makeflow.c

```diff
@@ -754,6 +761,11 @@ static int makeflow_check(struct dag *d)
 			if(skip_file_check || batch_fs_stat(remote_queue, f->filename, &buf) >= 0) {
 				continue;
 			}
+
+			if(f->source) {
```

Comment:
I keep getting:

Fails to parse the mountfile (example.mount)!

But as far as I can tell, my mount file follows your documentation... I'll send them to you by email.